### PR TITLE
Department admin

### PR DESCRIPTION
**Pull request by divs4debu**

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Create different component for both forms, not just one component containing both forms. And complete the code to float the course

**Comment by divs4debu** [comment]
@iamareebjamal  Please review

File: src/main/java/in/ac/amu/zhcet/controller/DepartmentController.java [hunk 1]
```diff
@@ -35,19 +44,21 @@ public String createCourse(@ModelAttribute Course course) {
...
 
-        return "redirect:department";
+        return "redirect:/department";
     }
 
     @PostMapping("/department/float_course")
-    public String floatCourse(@ModelAttribute Course course) {
+    public String floatCourse(@RequestParam String course, @RequestParam List<String> faculty) {
         // TODO: Add form handling
+        Course course1 = courseRepository.findByCode(course);
```
Comment (iamareebjamal) [inline src/main/java/in/ac/amu/zhcet/controller/DepartmentController.java]: Change this to course and param to courseCode

File: src/main/java/in/ac/amu/zhcet/controller/DepartmentController.java [hunk 2]
```diff
@@ -35,19 +44,21 @@ public String createCourse(@ModelAttribute Course course) {
...
+
         try {
-            departmentAdminService.floatCourse(course);
+            departmentAdminService.floatCourse(course1, faculty);
```
Comment (iamareebjamal) [inline src/main/java/in/ac/amu/zhcet/controller/DepartmentController.java]: Move `courseRepository.findByCode(course);` in departmentAdminService

All repository work should be done there

**Comment by divs4debu** [comment]
@iamareebjamal please review

**Pull request merged**